source/linear-algebra/source/01-LE/02.ptx

```diff
@@ -168,21 +182,28 @@ Add a constant to every term of a row, for example:
 <sidebyside>
     <p>
         <me>
-            \left[\begin{array}{cc|c}  1 &amp; 2 &amp; 3 
-                                    \\ 4 &amp; 5 &amp; 6 \end{array}\right]
+            \left[\begin{array}{cc|c}  1 &amp; 2 &amp; 4 
+                                    \\ 1 &amp; 3 &amp; 5 \end{array}\right]
         \sim
-            \left[\begin{array}{cc|c} 1+6 &amp; 2+6 &amp; 3+6
-                                    \\ 4 &amp; 5 &amp; 6 \end{array}\right]
```

Comment:
Actually adding a constant to rows in this example is perfectly fine (since the solution satisfies $x+y=1$ lol). 🤦‍♂️

New version does not satisfy $x+y=1$.